### PR TITLE
[CAZSM-256] bump glob parent and normalize-url

### PR DESCRIPTION
**Pull request by leusoijames**
## Checklist:

- [ ] It contains only changes required by issue (does not contain other PR)
- [ ] Includes link to an issue (if apply)
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Review by lukeaarm — APPROVED** [review]

**Comment by likeuwill** [comment]
It will not resolve the problem with audit warning, u can check it by running `yarn run improved-yarn-audit`. Also, u can check file `.iyarc` file by removing exclude number after u did changes and run `yarn run improved-yarn-audit` - warnings still exists. Yesterday was [merged](https://github.com/csstools/postcss-color-mod-function/pull/38#event-4887420800) fix for `1693` but we should wait to a new version of webpacker/rails.

Basically, we can't fix any warning because it's located inside `@rails/webpacker` library, so we should wait to a new version.
